### PR TITLE
ChasseurdeBots.Warsow version 2.1.2

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107155)

### CODE REVIEW

**Comment by Trenly** [comment]
> C:\Users\VALIDA~1\AppData\Local\Temp\is-3AKP9.tmp
2023-05-17 15:25:14.360   Message box (Yes/No):
                          The nickname field is empty or the default nick "Player", do you want to continue without setting a nickname?

image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230517230148

File: manifests/c/ChasseurdeBots/Warsow/2.1.2/ChasseurdeBots.Warsow.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ChasseurdeBots.Warsow
+PackageVersion: 2.1.2
+Installers:
+- InstallerUrl: https://warsow.net/warsow-2.1.2-setup.exe
+  Architecture: x64
+  InstallerType: inno
+  InstallerSha256: 937B8F62D1065CABAE5ABC22123B9C7E44BCC4A1AF55ED64B26F51D061F08D5D
```
Comment (stephengillie) [inline manifests/c/ChasseurdeBots/Warsow/2.1.2/ChasseurdeBots.Warsow.installer.yaml:10]:
```suggestion
  InstallerSha256: 937B8F62D1065CABAE5ABC22123B9C7E44BCC4A1AF55ED64B26F51D061F08D5D
  InstallerSwitches:
    Custom: /SuppressMsgBoxes
```

**Comment by stephengillie** [comment]
With the installer switch, the package installs in a Windows 11 VM normally, but errors when launched. I believe this error might be due to the VM having an inferior graphics card and driver.

> Faulting application name: warsow_x64.exe, version: 2.1.0.0, time stamp: 0x56f6b15f
Faulting module name: ref_gl_x64.dll_unloaded, version: 0.0.0.0, time stamp: 0x56f6b01f
Exception code: 0xc000041d
Fault offset: 0x0000000000047bb0
Faulting process id: 0x0x2164
Faulting application start time: 0x0x1D9893FC042A33F
Faulting application path: C:\Program Files (x86)\Warsow 2.1\warsow_x64.exe
Faulting module path: ref_gl_x64.dll
Report Id: 2ceae6b4-12df-403e-a93c-035ad9e72e89
Faulting package full name: 
Faulting package-relative application ID:

**Review by stephengillie — APPROVED** [review]

**Pull request merged**